Program.cs

```diff
@@ -60,7 +60,9 @@ static void Main(string[] args)
     <Assembly>
       <Parts>
         <Part id=""2"" partType=""Label-1"" position=""0,0,0"" rotation=""270,0,0"" drag=""0,0,0,0,0,0"" materials=""0,1,2"" scale=""1,1,1"" partCollisionResponse=""Default"" calculateDrag=""false"">
-           <Label.State designText = ""&lt;mspace={PixelSize}em&gt;&lt;line-height={PixelSize}em&gt;";
+           <Label.State designText = ""&lt";
+           
+           OutputContent += string.Format(CultureInfo.CreateSpecificCulture("en-US"), ";mspace={0:F2}em&gt;&lt;line-height={1:F2}em&gt;", PixelSize, PixelSize);

```

Comment:
CultureInfo requires "using System.Globalization" to be used
Also, CultureInfo.InvariantCulture is better than using en-US as the invariant culture won't be changed in the future